numpy/core/tests/test_nditer.py

```diff
@@ -2284,7 +2284,8 @@ def test_iter_nested_iters_dtype_copy():
     assert_equal(vals, [[0, 1, 2], [3, 4, 5]])
     vals = None
 
-    # updateifcopy
+    # writebackifcopy
+    # XXX ugly - is there a better way? np.nested_iter returns a tuple
```

Comment:
Why not `with i, j`?

Reply:
That's one possibility. I think people were looking for a one-line solution and Python 3 may require a two-line solution.

Reply:
I don't have a good enough understanding of `nested_iters` to know what makes sense here. Is it documented anywhere? Do the `nditer`s depend on each other?

Reply:
It is part of the public API, but it isn't documented.

Reply:
It would be nice if it was at least internally documented. Right now, I have very little idea of what it does.

Reply:
I would prefer to resolve the ``nested_iter`` issue in a separate pull request. I have filed issue #10009 a a reminder to do so. AFAICT the c-api function NpyIter_NestedIters is not part of the public API, so we could fix this on the python level only

Reply:
heh, in the end it is enough to do what @eric-wieser suggested in the first comment, simple and clear. I left the issue open since the function is not documented